[2022-12-03 01:43:25,555] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dag1.save_to_csv scheduled__2022-10-26T02:50:00+00:00 [queued]>
[2022-12-03 01:43:25,829] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dag1.save_to_csv scheduled__2022-10-26T02:50:00+00:00 [queued]>
[2022-12-03 01:43:25,837] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 01:43:25,957] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-12-03 01:43:25,957] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-12-03 01:43:26,861] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): save_to_csv> on 2022-10-26 02:50:00+00:00
[2022-12-03 01:43:27,102] {standard_task_runner.py:52} INFO - Started process 278 to run task
[2022-12-03 01:43:30,331] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'save_to_csv', 'scheduled__2022-10-26T02:50:00+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/AirflowClean2.py', '--cfg-path', '/tmp/tmpmw2wlmq9', '--error-file', '/tmp/tmpy3dkrds9']
[2022-12-03 01:43:30,918] {standard_task_runner.py:77} INFO - Job 70: Subtask save_to_csv
[2022-12-03 01:43:34,035] {logging_mixin.py:109} INFO - Running <TaskInstance: dag1.save_to_csv scheduled__2022-10-26T02:50:00+00:00 [running]> on host 422c372562ce
[2022-12-03 01:43:36,951] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Fadlil
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=save_to_csv
AIRFLOW_CTX_EXECUTION_DATE=2022-10-26T02:50:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-26T02:50:00+00:00
[2022-12-03 01:43:37,632] {python.py:175} INFO - Done. Returned value was: None
[2022-12-03 01:43:39,144] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=save_to_csv, execution_date=20221026T025000, start_date=20221203T014325, end_date=20221203T014339
[2022-12-03 01:43:39,655] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-12-03 01:43:40,099] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
